definitions/ext-flex_host/definition.yml

```diff
@@ -0,0 +1,30 @@
+domain: EXT
+type: FLEX_HOST
```

Comment:
wondering if this should be under `EXT-HOST` instead of it's own type.

Thoughts @srvaroa

Reply:
The metrics and dashboards included are very different, would this cause any issues? The current ext-host has DD based metrics and it looks quite limited in comparison.

Also if we were to do this, is there a good example I can use as a reference? thx

Reply:
Yeah shape of data is not a problem in this case, we can provide different representations.

Yo have an example here: https://github.com/newrelic/entity-definitions/blob/main/definitions/ext-service/definition.yml


This is my main question to move this PR forward so let me ping the team to have more input on how we should do it